plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveMetadata.java

```diff
@@ -386,6 +386,7 @@
     private final PartitionProjectionService partitionProjectionService;
     private final boolean allowTableRename;
     private final long maxPartitionDropsPerQuery;
+    private final HiveTimestampPrecision timestampPrecision;
```

Comment:
maybe call it viewsTimestampPrecision, since typically the timestampPrecision is taken from the session property.

Reply:
Decided to call it `hiveViewsTimestampPrecision` in line with `hiveViewsRunAsInvoker` field from `HiveMetadata`.